dali/dfuplus/dfuplus.cpp

```diff
@@ -1448,7 +1452,262 @@ int CDfuPlusHelper::abort()
     {
         error("Aborted %s\n", wuid);
     }
-    
+
+    return 0;
+}
+
+int CDfuPlusHelper::listhistory()
+{
+    const char* lfn = globals->queryProp("lfn");
+    if(lfn == NULL || *lfn == '\0')
+            throw MakeStringException(-1, "srcname not specified");
+
+    Owned<IClientListHistoryRequest> req = dfuclient->createListHistoryRequest();
+    req->setName(lfn);
+
+    Owned<IClientListHistoryResponse> resp = dfuclient->ListHistory(req);
+
+    const IMultiException* excep = &resp->getExceptions();
```

Comment:
It is legacy, I was just commenting, not suggesting change now. It's also quite wide spread.
If it were to change, probably something for @afishbeck or @wangkx